taskfiles/utils.yml

```diff
@@ -19,6 +19,9 @@ tasks:
         --numeric-owner
         --owner 0
         --sort name
+        {{- range .EXCLUDE_PATHS}}
```

Comment:
Since this is an optional variable that may not be obvious to future users, we should probably document it. Initially, I was planning to document this in the docs but I think it's probably better to document it inline. So how about we document task parameters like this:

```yml
  # @param {string} DATA_DIR The directory to compute the checksum for.
  # @param {string} OUTPUT_FILE
  # @param {[]string} [EXCLUDE_PATHS] A list of paths to exclude from the checksum.
```

The docstring uses JSDoc's syntax except the types use Go's syntax. I don't think we need to document
 the return value since that's kind of nebulous for a task; and the description already has a dedicated field in the task.

Reply:
sounds good